### PR TITLE
LPS-139466 Sidebar is vanishing when mapping a fragment in DPT

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/StructureTree.js [hunk 1]
```diff
@@ -451,7 +451,7 @@ function getMappedFieldLabel(
 				selectedMappingTypes.type.id,
 				selectedMappingTypes.subtype.id || 0
 		  )
-		: getMappingFieldsKey(infoItem.classNameId, infoItem.classTypeId);
+		: getMappingFieldsKey(infoItem?.classNameId, infoItem?.classTypeId);
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/StructureTree.js:454]: With this solution the key is being generated as `undefined-undefined`, what is a bit weird to me.
Maybe we could just check if `infoItem` is true, call `getMappingFieldsKey` only in that case, and set key to null if not, what do you think? 😄

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**